python/tvm/runtime/ndarray.py

```diff
@@ -249,6 +249,20 @@ def copyto(self, target):
             return self._copyto(res)
         raise ValueError("Unsupported target type %s" % str(type(target)))
 
+    def create_view(self, shape):
+        shape_imm = []
```

Comment:
nit: shall we just ShapeTuple instead of raw pointer to np array?

Reply:
(Related PR #10728 made to update the `TVMArrayAllocWithScope` signature to also use `ShapeTuple`, as long as I'm thinking on it.)